tools/run_tests.py

```diff
@@ -1,312 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
currently soak cluster automation relies on run_tests.py.  CC: @susanxhuynh 
If we drop this, we'll have to rejigger soak automation at the same time.